Sources/Plot/API/HTML.swift

```diff
@@ -109,9 +111,12 @@ public protocol HTMLDimensionContext: HTMLContext {}
 /// Context shared among all HTML elements that act as some form
 /// of link to an external resource, such as `<link>` or `<a>`.
 public protocol HTMLLinkableContext: HTMLContext {}
-/// Context shared among all HTML elements that enable media playback,
+/// Context shared among all HTML elements that support the `media`
+/// attribute for example `<source>`
```

Comment:
Very nitpicky (and happy to fix post-merge): Add a comma before `for example`, and a period at the end of the sentence:
```
attribute, for example `<source>`.
```

Reply:
Will fix.